src/plugins/home/public/application/components/solutions_section/solution_title.tsx

```diff
@@ -40,26 +40,125 @@ import {
   EuiIcon,
   IconType,
 } from '@elastic/eui';
+import { HomePluginBranding } from '../../../plugin';
```

Comment:
nit: is there a better route for this?

Reply:
Decided to put the import statements of type Branding in plugin.ts because most imports are collected there. There are no shorter path to import that.